doc/source/myrocks/variables.rst

```diff
@@ -628,6 +663,17 @@ until there is less than 10 bits of free space remaining.
 
 Allowed range is from ``1`` to ``2147483647``.
 
+.. variable:: rocksdb_bulk_load_allow_unsorted
+
+  :version 5.7.20-18: Implemented
+  :cli: ``--rocksdb-bulk-load-allow-unsorted``
+  :dyn: Yes
+  :scope: Global, Session
+  :vartype: Boolean
+  :default: ``OFF``
+
+When enabled input is sorted by Primary Key during bulkload.
```

Comment:
The variable name "allow unsorted" seems to be the opposite of the description "sort"?

Reply:
Considering the origination of this and the feature, this is correct. Previously, bulk load would only succeed if keys/rows were added in perfect sort order. Enabling this allows rows to be added in unsorted order but at a performance penalty. It's not ideal, but it _is_ correct in MyRocks/RocksDB semantics.

Reply:
I see, then a clarification is needed with what you wrote: when enabled, unsorted inputs are allowed, which are then sorted by the bulkloader itself, at a performance penalty